src/pages/home/report/ListBoundaryLoader/ListBoundaryLoader.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReportActionsSkeletonView from '../../../../components/ReportActionsSkeletonView';
+import CONST from '../../../../CONST';
+import useNetwork from '../../../../hooks/useNetwork';
+import ListHeaderComponentLoader from './ListHeaderComponentLoader/ListHeaderComponentLoader';
+
+const propTypes = {
+    type: PropTypes.string.isRequired,
```

Comment:
Please add JSDoc comments for all these props

Reply:
Also, let's make the `type` more specific:

```suggestion
    type: PropTypes.oneOf([CONST.LIST_COMPONENTS.HEADER, CONST.LIST_COMPONENTS.FOOTER]),
```

Reply:
👍🏻